Sources/KeyStore.swift

```diff
@@ -87,20 +87,21 @@ public final class KeyStore {
         guard let privateKey = PrivateKey(data: privateKeyData) else {
             throw Error.invalidKey
         }
-        return try self.import(privateKey: privateKey, password: newPassword)
+        return try self.import(privateKey: privateKey, password: newPassword, coin: key.coin ?? .ethereum)
```

Comment:
can you add `coin: Coin` into import function?

Reply:
It's there already, what do you mean?

Reply:
Line `80` `public func `import`(json: Data, password: String, newPassword: String) throws -> Wallet {` - this needs to take `coin: Coin`, because it would be imported from the iOS and I would be passing coin

Reply:
Same as you do for 
```swift
public func `import`(privateKey: PrivateKey, password: String, coin: Coin) throws -> Wallet
```

Reply:
ok